pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -406,7 +406,31 @@ func (a *ApplicationRestoreController) handle(ctx context.Context, restore *stor
 	}
 	switch restore.Status.Stage {
 	case storkapi.ApplicationRestoreStageInitial:
-		// Make sure the namespaces exist
+		if restore.Spec.BackupObjectType == resourcecollector.PxBackupObjectType_virtualMachine {
+			//if restore.Annotations[utils.PxBackupObjectType] == utils.PxBackupObjectType_virtualMachine {
```

Comment:
I am curious about why this line of code is commented here

Reply:
Thanks for pointing out. This is a miss.